lib/msal-common/src/config/Configuration.ts

```diff
@@ -39,6 +39,7 @@ export type Configuration = {
 export type AuthOptions = {
     clientId: string;
     authority?: string;
+    knownAuthorities?: Array<string>
```

Comment:
Wherever you set your defaults, this should be defaulted to an empty array

Reply:
Done.